client/src/components/App.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import '../assets/style.css';
+import data from '../fake-data/data';
```

Comment:
do you need this line if you already passed in the data as props?

Reply:
Thanks for the feedback! I'll get rid of it/plan to start with the mock data soon.